libpebble3/src/androidMain/kotlin/io/rebble/libpebblecommon/notification/LibPebbleNotificationListener.kt

```diff
@@ -156,7 +155,13 @@ class LibPebbleNotificationListener : NotificationListenerService(), LibPebbleKo
     // Note (see above comments), if onListenerConnected was called twice, then so will this be, for
     // *every* notification. So - the handler must be resilient to this.
     override fun onNotificationPosted(sbn: StatusBarNotification) {
+        if (!configHolder.value.sendNotifications) {
+            logger.v { "Notification from ${sbn.packageName} filtered - sendNotifications is off" }
```

Comment:
Anywhere that is logging package names should use a `PrivateLogger` to (optionally) obfuscate e.g. `${sbn.packageName.obfuscate(privateLogger)}`

Reply:
Updated